tests/scalers/metrics_api/metrics_api_test.go

```diff
@@ -162,9 +162,10 @@ spec:
       targetValue: "5"
       activationTargetValue: "20"
       url: "{{.MetricsServerEndpoint}}"
-      valueLocation: 'value'
-      authMode: "basic"
-      method: "query"
+      valueLocation: value
+      format: json
```

Comment:
does this mean users will need to default already existing metrics-api triggers with `format`? We should strive towards a code where the tests pass without this change imho.

Reply:
I've reverted that change